src/main/scala/mesosphere/marathon/raml/AppConversion.scala

```diff
@@ -79,7 +79,8 @@ trait AppConversion extends DefaultConversions with ConstraintConversion with En
       versionInfo = app.versionInfo.toRaml,
       unreachableStrategy = Some(app.unreachableStrategy.toRaml),
       killSelection = app.killSelection.toRaml,
-      tty = app.tty
+      tty = app.tty,
+      executorResources = app.executorResources.toRaml
```

Comment:
I think you need a similar update for `appProtoRamlWriter` (used for migrations)